net.sourceforge.tomatoes.IHaveNoTomatoes.appdata.xml

```diff
@@ -0,0 +1,47 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2022 Stephan Sokolow -->
+<component type="console-application">
+  <id>net.sourceforge.tomatoes.IHaveNoTomatoes</id>
+  <metadata_license>MIT</metadata_license>
+  <project_license>Zlib</project_license>
+  <name>I Have No Tomatoes</name>
+  <summary>Tomato smashing game</summary>
+  <developer_name>Mika Halttunen</developer_name>
+  <description>
+    <p>I Have No Tomatoes is an extreme leisure time activity idea of which culminates in the following question: How many tomatoes can you smash in ten short minutes? If you have the time to spare, this game has the vegetables just waiting to be eliminated!</p>
+  </description>
+  <launchable type="desktop-id">net.sourceforge.tomatoes.IHaveNoTomatoes.desktop</launchable>
+  <categories>
+    <category>Game</category>
+    <category>ArcadeGame</category>
+  </categories>
+  <icon type="local" width="64" height="64">/app/share/icons/hicolor/64x64/apps/net.sourceforge.tomatoes.IHaveNoTomatoes.png</icon>
+  <icon type="remote" width="51" height="51">http://tomatoes.sourceforge.net/tomato.gif</icon>
```

Comment:
```suggestion
```

Reply:
The Appstream XML docs seemed to imply that more was better on that front and, if the Flathub docs disagreed, I must have missed it.

Would you mind clarifying what heuristic I'm supposed to be applying?

Reply:
appstream compose will deal with it based in the .desktop file